libs/common/src/dtos/announcement.dto.ts

```diff
@@ -35,6 +35,10 @@ export class UpdateDto {
   @IsEnum(UpdateAnnouncementTypeDto)
   targetAnnouncementType: UpdateAnnouncementTypeDto;
 
+  @IsString()
```

Comment:
`UpdateDto` should not have `inReplyTo` based on what we have in spec.

Reply:
Ok let me check this, then a update to reply is update to self anyways right?

Reply:
an Update can get applied to Reply or Broadcast types. So yes